deploy/008-configure-acm-commands-aggregator.ts

```diff
@@ -0,0 +1,691 @@
+    ...getProtocolShareReservePermissions(SEPOLIA_PSR),
+    ...getConverterNetworkPermissions(SEPOLIA_CONVERTER_NETWORK),
+    ...getComptrollerPermissions(),
+    ...getVTokenPermissions(),
+    ...getRewardDistributorPermissionsBlockbased(),
+    ...getIRMPermissions(),
+    ...getConverterPermissions(),
+    ...getXVSVaultTreasuryPermissions(SEPOLIA_XVS_VAULT_TREASURY),
+  ],
+  opbnbtestnet: [
+    ...getResilientOraclePermissions(OPBNBTESTNET_RESILIENT_ORACLE),
+    ...getResilientOraclePermissions(OPBNBTESTNET_BOUND_VALIDATOR),
+    ...getBinanceOraclePermissions(OPBNBTESTNET_BINANCE_ORACLE),
+    ...getXVSPermissions(OPBNBTESTNET_XVS),
+    ...getXVSBridgeAdminPermissions(OPBNBTESTNET_XVS_BRIDGE_ADMIN),
+    ...getXVSVaultPermissions(OPBNBTESTNET_XVS_VAULT_PROXY),
+    ...getPoolRegistryPermissions(OPBNBTESTNET_POOL_REGISTRY),
+    ...getProtocolShareReservePermissions(OPBNBTESTNET_PSR),
+    ...getComptrollerPermissions(),
+    ...getVTokenPermissions(),
+    ...getIRMPermissions(),
+  ],
+};
+
+const revokePermissions: Permissions = {
+  arbitrumone: [
+    ...getPrimeRevokePermissions(ARBITRUMONE_PRIME, ARBITRUMONE_GUARDIAN),
+    ...getPrimeLiquidityProviderRevokePermissions(ARBITRUMONE_PLP, ARBITRUMONE_GUARDIAN),
+    ...getResilientOracleRevokePermissions(ARBITRUMONE_RESILIENT_ORACLE, ARBITRUMONE_GUARDIAN),
+    ...getBoundValidatorRevokePermissions(ARBITRUMONE_BOUND_VALIDATOR, ARBITRUMONE_GUARDIAN),
+    ...getXVSVaultRevokePermissions(ARBITRUMONE_XVS, ARBITRUMONE_GUARDIAN),
+    ...getRewardDistributorRevokePermissions(ARBITRUMONE_GUARDIAN),
+    ...getIRMRevokePermissions(ARBITRUMONE_GUARDIAN),
+    ...getPoolRegistryRevokePermissions(ARBITRUMONE_POOL_REGISTRY, ARBITRUMONE_GUARDIAN),
+    ...getComptrollerRevokePermissions(ARBITRUMONE_GUARDIAN),
+    ...getVTokenRevokePermissions(ARBITRUMONE_GUARDIAN),
+  ],
+  ethereum: [
+    ...getPrimeRevokePermissions(ETHEREUM_PRIME, ETHEREUM_GUARDIAN),
+    ...getPrimeLiquidityProviderRevokePermissions(ETHEREUM_PLP, ETHEREUM_GUARDIAN),
+    ...getResilientOracleRevokePermissions(ETHEREUM_RESILIENT_ORACLE, ETHEREUM_GUARDIAN),
+    ...getBoundValidatorRevokePermissions(ETHEREUM_BOUND_VALIDATOR, ETHEREUM_GUARDIAN),
+    ...getXVSVaultRevokePermissions(ETHEREUM_XVS, ETHEREUM_GUARDIAN),
+    ...getRewardDistributorRevokePermissions(ETHEREUM_GUARDIAN),
+    ...getIRMRevokePermissions(ETHEREUM_GUARDIAN),
+    ...getPoolRegistryRevokePermissions(ETHEREUM_POOL_REGISTRY, ETHEREUM_GUARDIAN),
+    ...getComptrollerRevokePermissions(ETHEREUM_GUARDIAN),
+    ...getVTokenRevokePermissions(ETHEREUM_GUARDIAN),
+    ...getRedstoneOracleRevokePermissions(ETHEREUM_REDSTONE_ORACLE, ETHEREUM_GUARDIAN),
+    ...getConverterNetworkRevokePermissions(ETHEREUM_CONVERTER_NETWORK, ETHEREUM_GUARDIAN),
+    ...getSFrxETHOracleRevokePermissions(ETHEREUM_sFrxETH_ORACLE, ETHEREUM_GUARDIAN),
+    ...getConvertersRevokePermissions(ETHEREUM_CONVERTERS, ETHEREUM_GUARDIAN),
```

Comment:
I think `fundXVSVault(uint256)` permission should also be revoke from guardian in XVSVaultTreasury